simulator/src/simulator/misc.ts

```diff
@@ -9,6 +9,9 @@ export const MINUTE = 60 * SECOND
 export const ORE = 1
 export const IRON = 100000000 * ORE
 
+/** 1 / 100000000 */
```

Comment:
This is backwards right? When I want to convert IRON TO ORE I would multiply 10 IRON * IRON_TO_ORE = 1000000000 ORE